examples/mcp_servers/authorization/src/authorization/server.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
+"""authorization MCP server"""
+
+from typing import Annotated
+
+import httpx
+from arcade_mcp_server import Context, MCPApp
+from arcade_mcp_server.auth import Reddit
+from arcade_mcp_server.resource_server import (
+    AccessTokenValidationOptions,
+    AuthorizationServerEntry,
+    ResourceServer,
+)
+
+# Option 1: Single authorization server example
+resource_server = ResourceServer(
+    canonical_url="http://127.0.0.1:8000/mcp",
+    authorization_servers=[
+        AuthorizationServerEntry(  # WorkOS Authkit example configuration
+            authorization_server_url="https://your-workos.authkit.app",
+            issuer="https://your-workos.authkit.app",
+            jwks_uri="https://your-workos.authkit.app/oauth2/jwks",
+            validation_options=AccessTokenValidationOptions(verify_aud=False),
```

Comment:
Is there a reason we're showing `verify_aud=False` here? Ideally we would do audience verification - see [9.2 Token Handling](https://modelcontextprotocol.io/specification/2025-11-25/basic/authorization#token-handling)